src/main/kotlin/org/jetbrains/exposed/dao/UUIDEntity.kt

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.exposed.dao
+
+import org.jetbrains.exposed.sql.Column
+import java.util.UUID
+
+open class UUIDTable(name: String = "", columName: String = "id") : IdTable<String>(name) {
+  override val id: Column<EntityID<String>> = varchar(columName, 36)
```

Comment:
I see. Well, in that case I think that will need a solution before this get merged, but I'm just a user of this project, so take this with a grain of salt :).

My only reasoning is that if I see a `UuidTable` that is part of the API and I use it, I will end up with the wrong type depending on the DB I use, which is surprising and the wrong thing to do.